package.json

```diff
@@ -42,6 +42,7 @@
     "babel-loader": "^8.0.6",
     "browserslist-config-openmrs": "^1.0.0",
     "clean-webpack-plugin": "^3.0.0",
+    "css-loader": "^3.2.0",
```

Comment:
If using css loader and css modules, maybe we should move away from `@emotion`? I think css modules will be easier for people not used to react.

W don't have to make a switch away from emotion in this PR, but we should probably only use one css lib per repo and should move over at some point.

Reply:
I can do the switch on this PR to avoid the technical debt.